atomone-1/README.md

```diff
@@ -0,0 +1,75 @@
+# 🔗 `atomone-1`
+
+![chain-id](https://img.shields.io/badge/chain%20id-atomone--1-blue?style=for-the-badge)
+![version: v1.0.0](https://img.shields.io/badge/version-v1.0.0-green?style=for-the-badge)
+<!-- ![genesis-time](https://img.shields.io/badge/%E2%8F%B0%20genesis%20time-2024--02--27T13%3A00%3A00Z-red?style=for-the-badge) -->
+
+## Proposed base genesis file
+
+The candidate base genesis file resulting from the currently on-chain govgen proposals [1](https://app.govgen.io/proposals/1) and [3](https://app.govgen.io/proposals/3) is available at https://atomone.fra1.digitaloceanspaces.com/genesis.json
+
+This genesis file is of course missing the validators gentxs, which will need to be collected and added to the genesis to be able to launch the chain.
+
+ For more information on how the genesis was built please also look [here](https://github.com/atomone-hub/govbox/blob/master/PROP-001.md).
+
+## Register in the Genesis
+
+To register your validator node in the `genesis.json` you will need to provide a signed `gentx` with an initial delegation of at least `1000000uatone`.
+To create your own genesis transaction (`gentx`) you will have to choose the following parameters for your validator: `commission-rate` (>=0.05), `commission-max-rate`, `commission-max-change-rate`, `min-self-delegation` (>=1), `website` (optional), `details` (optional), `identity` ([keybase](https://keybase.io) key hash, used to get validator logos in block explorers - optional), `security-contact` (email - optional).
+
+The `commission-rate`, `commission-max-rate`, `commission-max-change-rate` are free to set to the value you want, this values are just for example, note that 0.05 is 5%.
+
+```sh
+# Create the gentx
+atomoned genesis gentx your-key-name 1000000uatone \
+  --node-id $(atomoned tendermint show-node-id) \
+  --chain-id atomone-1 \
+  --commission-rate 0.05 \
+  --commission-max-rate 0.1 \
+  --commission-max-change-rate 0.05 \
+  --min-self-delegation 1 \
+  --website "https://foo.network" \
+  --details "My validator" \
+  --identity "id-from-keybase" \
+  --security-contact "security@foo.network"
+```
+
+## Operate the node
+
+### Install the binary
+
+- You can install the proposed chain binary from github release page
+
+https://github.com/atomone-hub/atomone/releases/tag/v1.0.0
+
+- Build from the source
+
+You need to have [go](https://go.dev/doc/install) installed
```

Comment:
Should it mentions `go version 1.21.x` here as this is not the latest release.

Reply:
You can easily downgrade to a lower version from the latest release of `go`, and the instructions to do so are printed by the `make` command of AtomOne if necessary. So I think the instructions here are sufficient.